src/components/TagWall/TagWall.js

```diff
@@ -71,7 +74,9 @@ const TagWall = ({
             removeBtnLabel={TAG_WALL_REMOVE_BUTTON}
             type="gray"
           >
-            {itemToString(item)}
+            <span className={`${carbonPrefix}text-truncate--end`}>
```

Comment:
Awesome! If we're applying the Carbon truncation helper class here, we should also import their partial (I believe it's called `globals/scss/helper-classes`) in `TagWall/_index.scss` to ensure that the styles will be compiled for individual component imports.

Reply:
Thanks! I think I'll keep using their helper class here (with the correct import of course). Otherwise I would need to still keep the `span` & create our own class to apply the `text-overflow` mixin to. Just seems easier this way.